pkg/steps/new_e2e.go

```diff
@@ -181,6 +199,50 @@ func imageForStep(image string, params api.Parameters) (string, error) {
 	return image, nil
 }
 
+func addSecretMount(pod *coreapi.Pod) {
+	path := "/tmp/installer"
```

Comment:
this seems awfully specific

Reply:
I am having a really hard time with naming, suggestions highly welcome.